src/java/org/streamingpool/core/service/impl/LocalPool.java

```diff
@@ -50,7 +54,9 @@
 public class LocalPool implements DiscoveryService, ProvidingService, StreamFactoryRegistry {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LocalPool.class);
+    private static final int DEFAULT_THREAD_POOL_SIZE = 100;
```

Comment:
as discussed, I'll remove default constructor for the LocalPool, so you must provide a Scheduler (anyway you should not instantiate the pool directly, but using Spring instead)